internal/hud/server/server.go

```diff
@@ -140,3 +146,44 @@ func (s *HeadsUpServer) HandleSail(w http.ResponseWriter, req *http.Request) {
 	}
 
 }
+
+func (s *HeadsUpServer) HandleTrigger(w http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodPost {
```

Comment:
Since the input is so simple you could also structure this API like:

```
GET /api/trigger/MANIFEST_NAME
```

And parse the URL for the manifest name. Up to you though!

Reply:
oh shit wait, you're making me realize that i did it this way specifically because i wanted it to be easier to trigger multiple manifests at once (e.g. if we have a button for "trigger all dirty") -- i guess by THAT logic, i should keep it as a payload and make it take a _list_ of manifest names...